src/assets/icons/tavlaLogo.tsx

```diff
@@ -1,8 +1,33 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import { colors } from '@entur/tokens'
+import { useSettingsContext } from '../../settings'
+
+function TavlaLogo({ className, height }: Props): JSX.Element {
+    const [settings] = useSettingsContext()
+    const [tavlaColor, setTavlaColor] = useState<string>(colors.brand.coral)
+    const [enturColor, setEnturColor] = useState<string>(colors.brand.blue)
+    const [underlineColor, setUnderlineColor] = useState<string>(
+        colors.brand.coral,
+    )
+
+    useEffect(() => {
+        if (settings) {
```

Comment:
Smak og behag, men for å unngå eit indenteringsnivå kan du snu om på det og "short-circuite" her:
```
if (!settings) return

if (settings.theme === 'dark') {
  ...